.pre-commit-config.yaml

```diff
@@ -14,5 +14,3 @@ repos:
         args:
           - --args=--config=__GIT_WORKING_DIR__/.tflint.hcl
       - id: terraform_checkov
-        args:
-          - --args=--config-file=__GIT_WORKING_DIR__/.checkov.yaml

```

Comment:
removing as we don't have a checkov file in a repo or pinned to a specific version. We need to do the major update for checkov...